manager/src/test/java/com/redhat/service/smartevents/manager/api/user/BridgesAPITest.java

```diff
@@ -134,6 +145,28 @@ public void getBridgeWithErrorHandler() {
         assertThat(retrievedBridge.getEndpoint()).isEqualTo(bridge.getEndpoint());
         assertThat(retrievedBridge.getErrorHandler()).isNotNull();
         assertThat(retrievedBridge.getErrorHandler()).isEqualTo(errorHandler);
+        assertThat(retrievedBridge.getCloudProvider()).isEqualTo(DEFAULT_CLOUD_PROVIDER);
+        assertThat(retrievedBridge.getRegion()).isEqualTo(DEFAULT_REGION);
+    }
+
+    @Test
+    @TestSecurity(user = DEFAULT_CUSTOMER_ID)
+    public void createBridge_withInvalidCloudProvider() {
+        ErrorsResponse errorsResponse = TestUtils.createBridge(new BridgeRequest(DEFAULT_BRIDGE_NAME, "dodgyCloudProvider", DEFAULT_REGION))
+                .as(ErrorsResponse.class);
+
+        ErrorResponse errorResponse = errorsResponse.getItems().get(0);
+        assertThat(errorResponse.getCode()).isEqualTo(errorDAO.findByException(InvalidCloudProviderException.class).getCode());
+    }
+
+    @Test
+    @TestSecurity(user = DEFAULT_CUSTOMER_ID)
+    public void createBridge_withInvalidRegion() {
+        ErrorsResponse errorsResponse = TestUtils.createBridge(new BridgeRequest(DEFAULT_BRIDGE_NAME, DEFAULT_CLOUD_PROVIDER, "dodgyRegion"))
+                .as(ErrorsResponse.class);
+
+        ErrorResponse errorResponse = errorsResponse.getItems().get(0);
+        assertThat(errorResponse.getCode()).isEqualTo(errorDAO.findByException(InvalidRegionException.class).getCode());
     }
```

Comment:
Do we want to assert also `errorsResponse.getItems().size()`? Asking as I saw @manstis  and you discussed the amount of constraint violation returned.